changelogs/current.yaml

```
@@ -5,6 +5,9 @@ behavior_changes:
 
 minor_behavior_changes:
 # *Changes that may cause incompatibilities for some users, but should not for most*
+- area: ext_authz
+  change: |
+    removing any query parameter in the presence of repeated query parameter keys no longer drops the repeats
```

Format checker wants dot at the end of the sentence. https://dev.azure.com/cncf/envoy/_build/results?buildId=144920&view=logs&j=c7de00aa-b0fb-52ba-707b-da6afd918eda&t=2ec0901d-43fb-521a-88ec-473706b56eab&l=32

```suggestion
    removing any query parameter in the presence of repeated query parameter keys no longer drops the repeats.
```